pkg/services/provisioning/provisioning.go

```diff
@@ -18,12 +18,11 @@ import (
 )
 
 type ProvisioningService interface {
```

Comment:
The whole point of this was to hide the Provisioners, in order to make the ProvisioningService less subject to evolutions